client/apps/edit/components/header/index.coffee

```diff
@@ -51,8 +51,11 @@ module.exports = class EditHeader extends Backbone.View
 
   save: (e) ->
     e.preventDefault()
-    @$('#edit-save').text 'Saving...'
-    @article.trigger('finished').save()
+    @$('#edit-save').text('Saving...').removeClass('attention')
+    if @article.get('published')
+      @article.trigger('savePublished').trigger('finished')
```

Comment:
Hmm yeah there's probably an opportunity to clean up the "autosave" events being triggered around here, but I don't mind the explicitness of this for now.